mods/creative/init.lua

```diff
@@ -54,6 +54,7 @@ if creative_mode_cache then
 				snappy  = caps,
 				choppy  = caps,
 				oddly_breakable_by_hand = caps,
+				dig_immediate = caps,
```

Comment:
```suggestion
				dig_immediate = {times = {nil, digtime, 0}, uses = 0, maxlevel = 256},
```
(Copied and modified from line 41.)

Reply:
I'd prefer `times = {[2] = digtime, [3] = 0}` because its more explicit